### DISCUSSION TITLE
How to select children?

### DISCUSSION THREAD

**Discussion by mwxxhdb** [ofajs/discussion#4] [Q&A] [answered]
The `$` will select only one element, the `all` will select multiple level elements, I'd like to know if there is a method to select only one level children element.

```html
<ul>
    <li>1</li>
    <li>2</li>
    <li>
		3
		<ul>
			<li>3-1</li>
			<li>3-2</li>
		</ul>
	</li>
</ul>
```

How to select 1, 2, 3 except 3-1 and 3-2?

**Comment by kirakiray** [comment, marked as answer]
Just get their parent element, which is itself an Array Like element. You can try this: `$('ul').forEach(e=>console.log(e.ele))`

https://ofajs.com/en/docs/get-started/basic-concept.html#Getting-Child-Elements-title2

`Array.from($('ul'))` or `$('ul').slice()`

**Comment by kirakiray** [comment]
new link: https://ofajs.com/en/api/operation/array-like
